File: android/OtaHandling/src/main/java/com/everysight/sdk/samples/otahandling/OtaHandlingScreen.kt
package com.everysight.sdk.samples.otahandling

import UIKit.app.Screen
import UIKit.app.data.Align
import UIKit.app.data.EvsColor
import UIKit.app.resources.Font
import UIKit.widgets.Text


class OtaHandlingScreen:Screen() {
    private val text = Text()
    override fun onCreate() {
        super.onCreate()
        text.setText("OTA Handling").setResource(Font.StockFont.Medium).setTextAlign(Align.center)
        text.setX(getWidth()/2).setY(getHeight()/2).setForegroundColor(EvsColor.Green.rgba)
        add(text)
    }
}

File: ios/OtaHandling/OtaHandling/Glasses/OtaHandlingScreen.swift
//Copyright (c) 2020 Everysight LTD

import Foundation
import NativeEvsKit

class OtaHandlingScreen : Screen {
    private let text = Text()
    override func onCreate() {
        super.onCreate()
        text.setText(text: "OTA Handling")
            .setResource(font: Font(stockFont: .medium))
            .setTextAlign(align: Align.center)
        text.setX(x: getWidth()/2)
            .setY(y: getHeight()/2)
            .setForegroundColor(color: EvsColor.green.rgba)
        add(uiElement: text)
    }
}
